index.js

```diff
@@ -22,11 +22,19 @@ program
   .option('-o, --output [PATH]', 'Directory where output will be stored.')
   .option('-b, --basic', 'Basic mode outputs only final image')
   .option('-c, --config [Object]', 'Options for the step')
+  .option('-l, --list', 'To list all the avaliable modules.')
   .option('--save-sequence [string]', 'Name space separated with Stringified sequence')
   .option('--install-module [string]', 'Module name space seaprated npm package name')
   .parse(process.argv);
 
-if (program.saveSequence) saveSequence(program, sequencer);
+if(program.list) {
+  var infoList = sequencer.modulesInfo();
+  for(var iL in infoList) {
```

Comment:
You can use `for(let info of infoList) {}`, of will the get the value instead of index.

Also, please rename `iL` to something more descriptive.